torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -280,8 +280,13 @@ NvrtcFunction nvrtcCompile(
 
 #ifdef __HIP_PLATFORM_HCC__
   std::vector<const char*> args = {"--std=c++14"};
+#else
+#if CUDA_VERSION == 11020
```

Comment:
Do we need to change those macros to ">="?

Reply:
I was hoping this is a one time thing. Let me double check it with those compiler guys.